templates/_macros/vf_linked-logo-section.jinja

```diff
@@ -0,0 +1,75 @@
+{#
+  Params
+  - title_text (string) (required): The text to be displayed as the heading
+  - layout: layout of linked logo section. Options are '50/50', '25/75', 'full-width'
+  - links (array) (required) : Array of links, each including 'href', 'label', 'text', and 'image_html'
+#}
+
+{% macro vf_linked_logo_section(title_text, links, layout="full-width") -%}
```

Comment:
I'd suggest having a look at whitespace controls inside this macro if you have some time. There is some extra whitespace between some of the lines in the HTML rendered by this macro:

<img width="399" alt="Screenshot 2025-04-02 at 10 31 03" src="https://github.com/user-attachments/assets/9dc4cd6b-23b0-44b2-a19d-553231199aa9" />

It's not really that egregious though, so I'd consider fixing this a "nice-to-have" as people will primarily be consuming via macro.

To test out how the HTML is rendered, use the language switcher at the top right of an example iframe to switch to HTML
<img width="399" alt="Screenshot 2025-04-02 at 10 33 43" src="https://github.com/user-attachments/assets/62532cfb-1b85-4f34-8a62-63c7724e53ad" />